.github/workflows/build_and_push_to_docker.yml

```diff
@@ -15,9 +15,6 @@ jobs:
           - ""
```

Comment:
for the un-versioned renku-python which corresponds to the latest `--pre`, is there an easy way to get the hash of the commit that renku is being installed from? e.g. can we call this version `dev[hash]`

Reply:
unfortunately not atm - we could change our versioning scheme to include tags, then they would be like `v0.10.3-19-gee950ec` (meaning 19 commits since v0.10.3, hash `ee950ec`)

Reply:
that would be cool 👍

Reply:
I can discuss it with the other renku-python people... though we are in favor of not publishing the dev releases to pypi anyway and just installing from git as needed

Reply:
huh -- i didn't think too hard about how the dev releases were available -- https://pypi.org/project/renku/#history > there's no way of getting this from pypi? we can't use the 0.10.4.dev14?

Reply:
I don't see an easy way atm - but I agree that we should fix this. Maybe we can open an issue?